pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/housekeeping/PubSubMessageBuilder.java

```diff
@@ -106,6 +106,14 @@ public PubsubMessage createMessage(String ddpMessageId,
                         if (jmlDtoOpt.isPresent()) {
                             userPreferredLangCode = jmlDtoOpt.get().getLanguageCode();
                         }
+                    } else if (queuedNotificationDto.getParticipantGuid() != null) {
```

Comment:
handle the scenario where Agedup users email is passed in the message during Ageup prep
During AGEUP prep , we collect aged-up users emailID and it is used to send Ageup related emails requesting to consent. User is not yet signed up on own but is a governed user and message will contain the user guid.
If the message contains participantGuid, try to get user preferred language to that emails can be sent in preferred language rather than study default.